### PR TITLE
Nicaea fixes and corrections

### PR DESCRIPTION
**Pull request by blrhc**

### CODE REVIEW

**Comment by blrhc** [comment]
> Good review!

Hi there @phibosGit! We're looking for people to participate in the project so definitely appreciate your enthusiasm here.
Might I suggest that you join our [Telegram group](https://t.me/JoyStreamOfficial) and we can discuss there how you might be able to contribute :)

**Comment by bedeho** [comment]
I concur, please join us!

**Comment by phibosGit** [comment]
Tks!!

Join in the group. Thanks for the $Welcome

@phibos
Josué Junior
Joinville - Santa Catarina

Em Sex, 31 de jul de 2020 14:26, Bedeho Mender <notifications@github.com>
escreveu:

File: README.md [hunk 1]
```diff
@@ -88,45 +90,41 @@ To exchange your tokens, follow these steps.
 5CktGSAEApx6Z2fK5NnK3dZNwD3Hi1YHhna3aTDHULSAXGdg
 ```
 
-Once the tokens have been received to this address, the time, date, your address, your memo and the current tJOY/USD exchange rate is logged. Your tokens are then burned (reducing the tJOY issuance), and the USD amount is deducted from the fiat pool. This means that the exchange rate is not affected. Your monero should arrive within 24 hours, as we are batching the transactions. Also note that the XMR/USD exchange rate is at the time of the monero transfer.
-
-In order for us to know what address to pay, you must tie your Joystream address to your monero address. The easiest way to do this is in the `My memo` tab under the `My Keys` sidebar.
+Once the tokens have been received to this address, the time, date, your address, your memo and the current tJOY/USD exchange rate are logged. Your tokens are then burned (reducing the tJOY issuance), and the USD amount is deducted from the fiat pool. This means that the exchange rate is not affected. Your Monero should arrive within 24 hours, as we are batching the transactions. Also note that the XMR/USD exchange rate is at the time of the Monero transfer.
```
Comment (bwhm) [inline README.md:93]: I have had a hard time keeping it up every day, so probably best to extend here. 72h?
```suggestion
Once the tokens have been received to this address, the time, date, your address, your memo and the current tJOY/USD exchange rate are logged. Your tokens are then burned (reducing the tJOY issuance), and the USD amount is deducted from the fiat pool. This means that the exchange rate is not affected. Your Monero should arrive within 72 hours, as we are batching the transactions. Also note that the XMR/USD exchange rate is at the time of the Monero transfer.
```

**Pull request merged**